04.ls/lib/l_option_formatter.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class LOptionFormatter
+  SECONDS_IN_HALF_YEAR = (365 / 2) * 24 * 60 * 60
+
+  class << self
+    def format(ls_files)
+      header = generate_header(ls_files)
+      body = generate_body(ls_files)
+      [header, *body].join("\n")
+    end
+
+    private
+
+    def generate_header(ls_files)
+      total_blocks = ls_files.map(&:blocks).sum
```

Comment:
```suggestion
      total_blocks = ls_files.sum(&:blocks)
```

でいけそう

あと変数名はtotalだけでも十分かも？
https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name